cmd/resources/resources.go

```diff
@@ -403,3 +403,21 @@ Additional help topics:{{range .Commands}}{{if .IsAdditionalHelpTopicCommand}}
 Use "{{.CommandPath}} [command] --help" for more information about a command.{{end}}
 `
 }
+
+func asMarkdown(s string) string {
+	renderer, _ := glamour.NewTermRenderer(
+		glamour.WithAutoStyle(),
+		glamour.WithWordWrap(0),
+		// remove default margin
```

Comment:
The default has a margin of 2 which looks a bit off with the surrounding margin-less text.